operations/research-and-development/product/technical-writing-team-handbook/writer-levels.md

```diff
@@ -0,0 +1,91 @@
+# Technial Writer Levels
+
+These levels serve to help Technical Writers assess their opportunities for career growth in the Techncial Writing function. The levels below follow a growth track in the art and craft of Technical Writing and not in a person management track.
+
+### Level 1: Technical Writer I
+
+| Leadership Principle | Competencies | Level Expectations |
+|---|---|---|
+| Ownership | Scope: "What do I work on?" <br> <br>*Scope is the domain over which you are expected to show ownership.* | Executes to deliver high quality documentation and in-product text updates. |
+| High-Impact | Supervision: "How much do I rely on my manager?"<br> <br>Community: "How do others work with me?"<br> <br>*The need to receive supervision is inversely related to the instinct to choose what is high-impact.* | Receives detailed instruction on most tasks. Asks appropriate questions, and knows when to stop and ask for help. <br><br>Works in cooperation with peers at the same level with ease. Utilizes the knowledge of others with more experience. |
+| Earn Trust | Influence: "Who do I work with?"<br> <br>*As more people and parts of the organization earn your trust, your influence grows.* | Works effectively in teams to align priorities and manage execution. |
+| Customer Obsession | Customer Impact: "How do I bring value?"<br> <br>*Community spans both open-source and enterprise, but ultimately manifests in the same obsession for enabling customers to be successful.* | Considers the customer's perspective when making decisions. Knows when to ask clarifying questions. |
+| Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration:  "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to work towards the best possible outcome and always iterating towards the best possible outcome.* | Regularly participates in team discussions for ongoing feature work.  May lead discussions on smaller topics.<br><br>Learns new skills and establishes goals and context quickly by asking precise questions. |
```

Comment:
```suggestion
| Self-Awareness | Interaction: "Who relies on me?"<br><br>Iteration: "How do I improve myself and others?"<br><br>*Self-awareness yields an ability to seamlessly fit into any conversation or effort, interacting with a group to iterate towards the best possible outcome.* | Regularly participates in team discussions for ongoing feature work. May lead discussions on smaller topics.<br><br>Learns new skills and establishes goals and context quickly by asking precise questions. |
```

Reply:
If you agree with this update, let's ensure all instances of this sentence are updated for consistency.

Reply:
This is another one that came from HR.  I will revert the commit.